lib/facilities/multi_client.rb

```diff
@@ -15,7 +16,7 @@ def run(requests)
       requests.map do |req|
         if req.response.timed_out?
           Rails.logger.error "GIS request timed out: #{req.url}"
-          raise Common::Client::Errors::RequestTimeout
+          raise Common::Exceptions::ClientError.new('Facility request timed out', {})
```

Comment:
I'm planning on refactoring this significantly to do local storage of facility data, so in the meantime I'm okay with bailing out of the loop if any of the sub-queries errors out. Since they are all going to the same arcgis endpoint I don't anticipate getting particularly useful differential errors back.